spaceship/lib/spaceship/test_flight/client.rb

```diff
@@ -187,7 +187,19 @@ def testers_for_app(app_id: nil)
         assert_required_params(__method__, binding)
         page_size = 40 # that's enforced by the iTC servers
         resulting_array = []
-        initial_url = "providers/#{team_id}/apps/#{app_id}/testers?limit=#{page_size}&sort=email&order=asc"
+        # Sort order is set to default, because any other sort order breaks
+        # pagination when dealing with lots of anonymous (public link) testers.
+        # All anonymous testers have the same first name, last name, and email,
```

Comment:
Everything in this comment from this line can be in the PR description, and a link to the PR here for future reference. Better than this big comment here, sorry for the confusion I caused.